src/lib/protocols/proof/messages/RequestPresentation.ts

```diff
@@ -0,0 +1,46 @@
+import { Equals, IsString } from 'class-validator';
+import { AgentMessage } from '../../../agent/AgentMessage';
+import { MessageType } from './messages';
+import { Attachment } from '../../../utils/Attachment';
+import { Expose, Type } from 'class-transformer';
+
+/**
+ *  Interface for RequestPresentation
+ */
+export interface RequestPresentationData {
+  id?: string;
+  comment?: string;
+  attachments: Attachment[];
+}
+
+/**
+ * This class is used for RequestPresentation
+ *
+ * @see https://github.com/hyperledger/aries-rfcs/blob/master/features/0037-present-proof/README.md#request-presentation
+ */
+export class RequestPresentationMessage extends AgentMessage {
```

Comment:
Can you rename you file to be the same as the class name:
`RequestPresentation.ts` -> `RequestPresentationMessage.ts`

Reply:
you suggested to rename RequestPresentation, ->  ProposePresentation in  messages.ts file. (This variable is used to store value '= 'did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/present-proof/1.0/request-presentation'')
i think we can also rename as RequestPresentation -> ProposePresentationMessage.ts.
What do you think?

Reply:
currently changed : RequestPresentation.ts -> RequestPresentationMessage.ts